biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -80,14 +90,14 @@ export class AppController {
   }
 
   // TODO: Add API to send required info dispatch_finish pattern to NATS
-  @MessagePattern('dispatch_finish')
-  async dispatchLog(data: any) {
+  @MessagePattern(MQDispatch.SIM_HPC_FINISH)
+  async dispatchFinish(uuid: string) {
```

Comment:
Im still not understanding here. It seems that the monitoring cron job is not started until the HPC_Finish message is received. Should it not be when the job is started?

Reply:
The executes when an actual simulation finishes on HPC